storage/engine/keys.go

```diff
@@ -53,7 +52,7 @@ func StoreStatKey(stat proto.Key) proto.Key {
 // StoreStatusKey returns the key for accessing the store status for the
 // specified store ID.
 func StoreStatusKey(storeID int32) proto.Key {
-	return MakeKey(KeyStatusStorePrefix, proto.Key(strconv.FormatInt(int64(storeID), 10)))
+	return MakeKey(KeyStatusStorePrefix, encoding.EncodeUvarint(nil, uint64(storeID)))
```

Comment:
No, I removed the key from there because it only puts a status at the end of a scanner iteration.